package.json

```diff
@@ -103,6 +104,7 @@
     "fs-extra": "^1.0.0",
     "fuse.js": "^2.2.0",
     "glob": "^7.0.0",
+    "glslify": "^4.0.0",
```

Comment:
we'll need to add a `transforms` field to the package.json so users can bundle plotly.js w/o having to worry about `-t glslify`